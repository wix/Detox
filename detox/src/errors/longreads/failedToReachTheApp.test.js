const failedToReachTheApp = require('./failedToReachTheApp');

describe('failedToReachTheApp', () => {
  test('.evenThoughAppWasLaunched()', () => {
    expect(failedToReachTheApp.evenThoughAppWasLaunched()).toMatchSnapshot();
  });

  test('.maybeAppWasNotLaunched(action)', () => {
    expect(failedToReachTheApp.maybeAppWasNotLaunched({
      type: 'isReady',
      messageId: -1000
    })).toMatchSnapshot();
  });
});
