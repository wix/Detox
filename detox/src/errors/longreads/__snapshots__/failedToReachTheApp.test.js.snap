// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`failedToReachTheApp .evenThoughAppWasLaunched() 1`] = `
[DetoxRuntimeError: Detox can't seem to connect to the test app(s)!

HINT: 

The test app might have crashed prematurely, or has had trouble setting up the connection.
Refer to our troubleshooting guide, for full details: https://wix.github.io/Detox/docs/troubleshooting/running-tests#tests-execution-hangs 
]
`;

exports[`failedToReachTheApp .maybeAppWasNotLaunched(action) 1`] = `
[DetoxRuntimeError: Detox can't seem to connect to the test app(s)!

HINT: 

Have you forgotten to call 'device.launchApp()' in the beginning of your test?
Refer to our troubleshooting guide, for full details: https://wix.github.io/Detox/docs/troubleshooting/running-tests#tests-execution-hangs

---
The following package could not be delivered:

{
  type: 'isReady',
  messageId: -1000
}]
`;
